### PR TITLE
dbcon IsConnected Runtime Fix

### PR DESCRIPTION
**Pull request by Geevies**

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
!bugfix

File: code/modules/admin/verbs/warning.dm [hunk 1]
```diff
@@ -312,7 +312,7 @@
 	var/ecolor = "#e3e3e3"	//gray colour, expired = 1
 
 	establish_db_connection(dbcon)
-	if (!dbcon.IsConnected())
+	if (!dbcon?.IsConnected())
```
Comment (Arrow768) [inline code/modules/admin/verbs/warning.dm:315]:
```suggestion
	if (!establish_db_connection(dbcon))
```

File: code/modules/mob/abstract/new_player/menu.dm [hunk 2]
```diff
@@ -199,7 +199,7 @@
 
 /obj/screen/new_player/selection/polls/Initialize()
 	. = ..()
-	if(dbcon.IsConnected())
+	if(dbcon?.IsConnected())
```
Comment (Arrow768) [inline code/modules/mob/abstract/new_player/menu.dm:202]:
```suggestion
	if(establish_db_connection(dbcon))
```

File: code/modules/admin/verbs/warning.dm [hunk 3]
```diff
@@ -312,7 +312,7 @@
 	var/ecolor = "#e3e3e3"	//gray colour, expired = 1
 
 	establish_db_connection(dbcon)
```
Comment (Arrow768) [inline code/modules/admin/verbs/warning.dm:314]:
```suggestion
```

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**